runtime/cspell.json

```diff
@@ -0,0 +1,183 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [],
+  "ignoreWords": [
+    "anyfunc",
+    "argty",
+    "assemblyscript",
+    "bulkmemory",
+    "chainlist",
+    "cranelift",
+    "demangle",
+    "dereferencable",
+    "dylib",
+    "emtests",
+    "Emscripten",
+    "externref",
+    "funcenv",
+    "funcref",
+    "funcrefs",
+    "funcs",
+    "Ggas",
+    "Greenwasm",
+    "hashbrown",
```

Comment:
IMO crate names like `hashbrown`, `indexmap` or `wasmparser` (consult Cargo.toml) should be considered valid words project-wide (i.e. in top-level `cspell.json`).